collector/netclass_rtnl_linux.go

```diff
@@ -186,3 +209,62 @@ func (c *netClassCollector) getLinkModes() ([]*ethtool.LinkMode, error) {
 
 	return lms, err
 }
+
+// getSysfsAttributes reads attributes that are absent from netlink but provided
+// by sysfs.
+func getSysfsAttributes(links []rtnetlink.LinkMessage) (sysfs.NetClass, error) {
```

Comment:
This sysfs specific logic should go into the procfs module

Reply:
@discordianfish would you suggest that I split out the attribute-specific stuff from `parseNetClassIface()` into a separate function that only reads a given file, export that, and use it here? That would solve the problem I think. The issue with `NetClassByIface()` is that it parses all the attributes for a device, while here we only want a couple of them.

Reply:
Yes, perfect!

Reply:
@discordianfish So that and a follow-up got merged, and I've now updated this PR to use it. PTAL thanks!